### PR TITLE
xcm-v3 benchmarks, weights, fixes for bridge-hubs

### PR DESCRIPTION
**Pull request by bkontur**

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2190571

File: parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs [hunk 1]
```diff
@@ -41,10 +40,9 @@ use xcm_executor::{traits::WithOriginFilter, XcmExecutor};
 
 parameter_types! {
 	pub const KsmRelayLocation: MultiLocation = MultiLocation::parent();
-	pub const RelayNetwork: Option<NetworkId> = Some(NetworkId::Kusama);
+	pub const RelayNetwork: NetworkId = NetworkId::Kusama;
```
Comment (KiChjang) [inline parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs:43]: How did this even manage to compile? `AccountId32Aliases` for example requires this to be an `Option<NetworkId>`.

Comment (bkontur) [reply parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs:43]: hmm, interesting, checking, CI passed with almost latest polkadot's `gav-xcm-v3`

https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2190503
```
Compiling xcm-builder v0.9.33 (https://github.com/paritytech//polkadot?branch=gav-xcm-v3#4ccf61a6)
```

strange, for example contracts-rococo has `pub const RelayNetwork: Option<NetworkId> = None;` and others have `pub const RelayNetwork: NetworkId` and both compiles

maybe some blanket/macro hidden magic?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2194452

**Pull request merged**